conventions.md

```diff
@@ -35,12 +36,12 @@ Languages described as having a sibilant/non-sibilant distinction among coronal
 Retroflex clicks are represented by `‼` (==203C DOUBLE EXCLAMATION MARK==). Note that `ǃ` (==01C3==, the symbol representing (post)alveolar clicks in the IPA) is confusingly referred to as ==LATIN LETTER RETROFLEX CLICK== in the Unicode standard.
 
 ## Glottalization
-Sounds described as “glottalized” are marked with `ˀ` (==02C0 MODIFIER LETTER GLOTTAL STOP==), unless it is clear from context that either “ejective” or “creaky voicing” are the intended meaning. See the section on [Glottalization, laryngealization, and creaky voicing](#glottalization-laryngealization-and-creaky-voicing) for more details.
+Sounds described as “glottalized” are marked with `ˀ` (==02C0 MODIFIER LETTER GLOTTAL STOP==), unless it is clear from context that either “ejective” or “creaky voicing” are the intended meaning. See the section on [Glottalization, laryngealization, and creaky voicing](#glottalization,-laryngealization,-and-creaky-voicing) for more details.
```

Comment:
@drammock -- link doesn't work for me, but also doesn't fail completely -- it goes to the conventions home page :)